examples/qt.py

```diff
@@ -108,6 +108,8 @@ class MainWindow(QMainWindow):
     def __init__(self):
         # noinspection PyArgumentList
         super(MainWindow, self).__init__(None)
+        # Avoids crash when shutting down CEF (issue #360)
+        self.setAttribute(Qt.WA_DeleteOnClose, True)
```

Comment:
Have you tested this code with pyqt4 and pyqt5? If not then maybe we should execute it only after checking the `PYSIDE` constant variable.

Reply:
I've added the guard just in case